src/lycophron/config.py

```diff
@@ -24,21 +25,36 @@ class Defaults:
 
 
 class Config(dict):
-    def __init__(self, root_path, defaults: dict = None) -> None:
-        super().__init__(defaults or {})
+    def __init__(self, root_path, defaults: None) -> None:
+        self.defaults = defaults
```

Comment:
This allows some variables to be defaulted by the caller. 

E.g. `DB_URI` can be set to a specific path instead of the current directory.